README.md

```diff
@@ -0,0 +1,96 @@
+# Podman Desktop Extension Minimal Template
+
+<p align="center">
+  <img alt="Hello World" src="/images/helloselkie.png" width="50%">
+</p>
+
+## Overview
+
+This template provides a minimal template on how to build a Podman Desktop extension. More information can be found on our [official extension documentation](https://podman-desktop.io/docs/extensions) on how to further expand your extension.
+
+With this template, on activating a "Hello World!" dialog will appear.
+
+## Development
+
+To build and develop the extension, follow these steps:
+
+1. Clone the project or your fork:
+```sh
+$ git clone https://github.com/containers/podman-desktop-extension-minimal-template/
+```
+
+2. Run `npm install` to install all relevant packages:
+```sh
+$ npm install
+```
+
+3. Create a build:
+
+Creating a build will generate all required files for Podman Desktop to load the extension:
+
+```sh
+$ npm build
+```
+Optionally, you can also use `npm watch` to continuously rebuild after each change, without needing to re-run `npm build`:
+
+```sh
+$ npm watch
+```
+
+4. Load the extension temporarily within Podman Desktop:
+
+We will load the extension within Podman Desktop to test it. This requires cloning the [Podman Desktop repo](https://github.com/containers/podman-desktop):
+
+```sh
+$ git clone https://github.com/containers/podman-desktop
+```
+
+Navigate to the directory:
+
+```sh
+$ cd podman-desktop
+```
+
+Run the `npm install` command:
+
+```sh
+$ npm install
+```
+
+Load the extension using the `npm watch` command with an additional parameter to load the `backend` packaged data:
+
+```sh
+npm watch --extension-folder ../podman-desktop-extension-minimal-template
+```
```

Comment:
it might not be clear that this command should be executed from a podman desktop development folder ?

Reply:
previous step says to cd into the podman-desktop directory since we will need to run the npm install command anyways (see line 51: https://github.com/containers/podman-desktop-extension-minimal-template/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R51)

Reply:
I'm just assuming people will not read all the steps carefully :-)

they'll apply all commands from the current folder of the template